BlogWebsite/Views/Partials/Components.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Umbraco.Cms.Core.Models.Blocks.BlockListModel>
+@{
+    if (!Model.Any()) { return; }
+
+    foreach (var component in Model.Where(x => ((IComponentComposition)x.Content).IsComposedOf(ComponentComposition.ModelTypeAlias) && ((IComponentComposition)x.Content).Active))
```

Comment:
@MikeMijnheer Volgens mij is het prima hier hier door de components wordt geloopt. Het enige is dat het misschien mooier is om dit in een Controller te gaan doen, maar voor nu prima denk ik. 😄